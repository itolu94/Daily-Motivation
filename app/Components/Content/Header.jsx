import React, {Component} from 'react';
import $ from 'jquery';
import {Dropdown, NavItem, Button} from 'react-materialize';

// TODO Convert to functional component
export default class Header extends Component {
    constructor(){
        super();
    }
    render() {
        return (
            <div>
                <nav id='header'>
                    <div className="nav-wrapper">
                        <a href="#" className="brand-logo center">Daily-Motivation</a>
                        <ul id="nav-mobile" className="left hide-on-med-and-down">
                            <li><a onClick={this.props.changePage} href="#" name="registration">Registration</a></li>
                            <li><a onClick={this.props.changePage} href="#" name="delete-account">Delete Account</a></li>
                            <li><a onClick={this.props.changePage} href="#" name="contact">Contact</a></li>
                        </ul>
                        <ul className="sidenav" id="mobile-demo">
                            <Dropdown id="mobile-demo" trigger={
                                <a href="#" className="sidenav-trigger"><i className="material-icons">menu</i></a>
                            }>
                                <li><a onClick={this.props.changePage} href="#" name="registration">Registration</a></li>
                                <li><a onClick={this.props.changePage} href="#" name="delete-account">Delete Account</a></li>
                                <li><a onClick={this.props.changePage} href="#" name="contact">Contact</a></li>
                            </Dropdown>
                        </ul>
                    </div>
                </nav>
            </div>
        );
    }
};





