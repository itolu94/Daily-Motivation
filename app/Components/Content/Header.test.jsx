import React from 'react';
import { shallow } from 'enzyme'
import Header from './Pages/Contact/Header.jsx';


test("Header component", () => {
    const wrapper = shallow(<Header/>);
    expect(wrapper).toMatchSnapshot();
});
