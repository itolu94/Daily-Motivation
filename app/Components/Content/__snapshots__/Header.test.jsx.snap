// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component 1`] = `
<div>
  <nav
    id="header"
  >
    <div
      className="nav-wrapper"
    >
      <a
        className="brand-logo center"
        href="#"
      >
        Daily-Motivation
      </a>
      <ul
        className="left hide-on-med-and-down"
        id="nav-mobile"
      >
        <li>
          <a
            href="#"
            name="registration"
          >
            Registration
          </a>
        </li>
        <li>
          <a
            href="#"
            name="delete-account"
          >
            Delete Account
          </a>
        </li>
        <li>
          <a
            href="#"
            name="contact"
          >
            Contact
          </a>
        </li>
      </ul>
      <ul
        className="sidenav"
        id="mobile-demo"
      >
        <Dropdown
          id="mobile-demo"
          trigger={
            <a
              className="sidenav-trigger"
              href="#"
            >
              <i
                className="material-icons"
              >
                menu
              </i>
            </a>
          }
        >
          <li>
            <a
              href="#"
              name="registration"
            >
              Registration
            </a>
          </li>
          <li>
            <a
              href="#"
              name="delete-account"
            >
              Delete Account
            </a>
          </li>
          <li>
            <a
              href="#"
              name="contact"
            >
              Contact
            </a>
          </li>
        </Dropdown>
      </ul>
    </div>
  </nav>
</div>
`;
